shell/components/auth/login/ldap.vue

```diff
@@ -69,7 +69,7 @@ export default {
             v-model:value="password"
             type="password"
             :label="t('login.password')"
-            autocomplete="password"
+            autocomplete="new-password"
```

Comment:
Should this be `autocomplete="current-password"` instead? 

https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Attributes/autocomplete#new-password